### PR TITLE
ci: add GitHub actions workflow

### PR DESCRIPTION
**Pull request by blocktrron**

### CODE REVIEW

File: .github/workflows/generate.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+name: Generate
+on:
+  push:
+  pull_request:
+    types: [opened, synchronize, reopened]
```
Comment (NeoRaider) [inline .github/workflows/generate.yml:5]: Should we also add a scheduled run, e.g. once a day/week/?

File: .github/workflows/generate.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+        run: sudo apt-get install python3-setuptools && python3 -m pip install -r requirements.txt
+      - name: Create output directory
+        run: mkdir out
+      - name: Build
+        run: ./census-exporter.py out/gluon-census.prom
+      - name: Archive output
+        uses: actions/upload-artifact@v1
+        with:
+          name: census-output
+          path: out
```
Comment (NeoRaider) [inline .github/workflows/generate.yml:22]: Missing newline

File: .github/workflows/generate.yml (hunk 1 above)
Comment (blocktrron) [reply .github/workflows/generate.yml:5]: Added scheduled daily job.

File: .github/workflows/generate.yml (hunk 2 above)
Comment (blocktrron) [reply .github/workflows/generate.yml]: fixed

**Pull request merged**